lms/djangoapps/grades/models.py

```diff
@@ -380,6 +381,7 @@ class Meta(object):
         # (user_id) for course dashboard; explicitly declared as an index below
         unique_together = [
             ('course_id', 'user_id'),
+            ('date_passed', 'course_id'),
```

Comment:
Actually, we want to create an index so we can easily find all users in a course with a passing grade.